### PR TITLE
Тягунов -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by vtyagunov**

### CODE REVIEW

**Comment by vtyagunov** [comment]
@AndrewSmrnv @KalyanovDO Please, review this PR

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public void canConvertComplexNumberToString() {
+        ComplexNumber number = new ComplexNumber(7, 10);
+        System.out.println(number.toString());
+        assertEquals(number.toString(), "7.0 + 10.0i");
+    }
+
+    @Test
+    public void isNumberEqualsToItself() {
+        ComplexNumber number = new ComplexNumber(7, 10);
+        assertEquals(number.equals(number), true);
```
Comment (AndrewSmrnv) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: @VladimirTyagunov , use assertTrue() instead

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    @Test
+    public void canCompareWithEmptyObject() {
+        ComplexNumber number = new ComplexNumber(7, 10);
+        Object object = new Object();
+        assertEquals(number.equals(object), false);
```
Comment (AndrewSmrnv) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: @VladimirTyagunov , use assertFalse() instead

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (AndrewSmrnv) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: @VladimirTyagunov , please, remove the star import

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java (hunk 1 above)
Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Fixed.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java (hunk 2 above)
Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Fixed.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java (hunk 3 above)
Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Fixed.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        ComplexNumber number = new ComplexNumber();
+        assertEquals(number.toString(), "0.0 + 0.0i");
+    }
+
+    @Test
+    public void canConvertComplexNumberToString() {
+        ComplexNumber number = new ComplexNumber(7, 10);
+        System.out.println(number.toString());
+        assertEquals(number.toString(), "7.0 + 10.0i");
+    }
```
Comment (KalyanovDO) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Сheck function assertEquals(double expected, double actual)
Could you swap arguments?

**Review by KalyanovDO — CHANGES_REQUESTED** [review]

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java (hunk 4 above)
Comment (VladimirTyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: > Сhech function assertEquals(double expected, double actual, double delta)

@KalyanovDO, Did you mean Curch?

image

Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: > Сhech function assertEquals(double expected, double actual, double delta)

@KalyanovDO, Did you mean Church?

image

Comment (KalyanovDO) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: I've meant church
https://vistapointe.net/images/church-wallpaper-18.jpg

**Comment by AndrewSmrnv** [comment]
👍

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java (hunk 4 above)
Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: > Сheck function assertEquals(double expected, double actual, double delta)

assertEquals in those tests have only String arguments.

Comment (vtyagunov) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: > Could you swap arguments?

Done.

Comment (KalyanovDO) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Sorry, my fault.

**Comment by KalyanovDO** [comment]
:+1:

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package ru.unn.agile.ComplexNumber.model;
```
Comment (VitKonevskiy) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java]: Please, use low-case for package-naming

**Pull request merged**